### DISCUSSION TITLE
preview-tui - wezterm + tmux + imgcat not working

### DISCUSSION THREAD

**Discussion by marcoSven** [jarun/nnn#1856] [Q&A]
Wezterm tmux Kitty img protocol is currently not working see https://github.com/wez/wezterm/issues/4531

Wezterm imgcat is working in tmux but unfortunately I am not able to get it to work within nnn using  a tmux split pane

```sh
... 
   elif [ "$NNN_TERMINAL" = "tmux" ] && [ "$NNN_PREVIEWIMGPROG" = "imgcat" ]; then
        wezterm imgcat "$3" &
    elif [ "$NNN_TERMINAL" = "tmux" ]; then
...

```

Something is getting rendered but is invisible/blank 

The preview with wezterm imgcat is working fine in a wezterm non tmux pane.

**Comment by qrockz** [comment]
**wezterm** 20240203-110809-5046fc22  + **tmux** 3.4 + **nnn** 4.9 **preview-tui (using wezterm imgcat)**
![2024-mär-28--00-59-59_maim](https://github.com/jarun/nnn/assets/56853194/87d678ba-c868-431c-a2fd-634d0602623a)

**Reply by marcoSven** [reply]
What you shared in your screenshot is what I see using Kitty img protocol in and outside of nnn.

Here is what I see using wezterm imgcat in and outside nnn.


https://github.com/jarun/nnn/assets/3860298/ef30a34d-e657-4d7a-a873-5a0debce9a45
